frontend/scss/components/organisms/agenda.scss

```diff
@@ -33,6 +33,39 @@
     }
   }
 
+  &-selector {
+
+    &-button {
+      @include hl;
+      @include hl-h3;
+      padding: 10px 30px 15px;
+      margin-bottom: 0;
+      flex-grow: 1;
+      outline: none;
+      color: fade_out(color('limed-spruce'), 0.57);
+      background: color('athens-gray');
+
+      &+.#{organism('agenda')} {
+        display: none;
+      }
+
+      &[option][selected] {
```

Comment:
Does that selector need to be double specific by using `[option]` and `[selected]` or would the later one be sufficient?

Reply:
amp-selector adds some default stylings to selected options using this double selector and without the double selector some of our styling properties are overridden (e.g. border: 'none')